Dockerfile.dev

```diff
@@ -24,11 +24,29 @@ RUN apt-get update -qq && apt-get install -y chromium \
   && rm -rf /var/lib/apt/lists/*
 
 # Node and Yarn
-RUN wget --no-check-certificate -qO - https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
-  && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
-  && curl -sL https://deb.nodesource.com/setup_19.x | bash - \
-  && apt-get update -qq && apt-get install -y --no-install-recommends nodejs yarn netcat \
-  && rm -rf /var/lib/apt/lists/*
+# OLD
+#RUN wget --no-check-certificate -qO - https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+#  && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
+#  && curl -sL https://deb.nodesource.com/setup_20.x | bash - \
+#  && apt-get update -qq && apt-get install -y --no-install-recommends nodejs yarn netcat \
+#  && rm -rf /var/lib/apt/lists/*
+
+# apparently this conflicts with yarn?
+# https://github.com/yarnpkg/yarn/issues/2821
+#RUN apt-get update -qq && apt-get remove cmdtest \
+#  && rm -rf /var/lib/apt/lists/*
+
+# https://github.com/nodesource/distributions#debian-and-ubuntu-based-distributions
+RUN apt-get update -qq \
+  && apt-get install -y ca-certificates curl gnupg \
+  && mkdir -p /etc/apt/keyrings \
+  && curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg \
+  && NODE_MAJOR=20 \
+  && echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | tee /etc/apt/sources.list.d/nodesource.list \
+  && apt-get update -qq \
+  && apt-get install -y --no-install-recommends nodejs
```

Comment:
is `yarn` not installable via apt-get?

also there should be a `&& rm -rf /var/lib/apt/lists/*` line at the end of this still

Reply:
i was getting werid stuff with apt-get, and then found a post on stackoverflow saying "Just use npm"...